pontoon/contributors/tests.py

```diff
@@ -260,9 +262,9 @@ def test_given_period(self):
         Checks if view sets and returns data for right period.
         """
         with patch(
-            'django.utils.timezone.now',
-            wraps=now,
-            return_value=aware_datetime(2015, 7, 5)
+                'django.utils.timezone.now',
+                wraps=now,
+                return_value=aware_datetime(2015, 7, 5)
```

Comment:
Why did you add that level of indentation? It shouldn't be required. If PEP8 does require it, then instead you should write it like this: 

```python
with (
    patch(
        'django.utils.timezone.now',
        # ...
    )
):
```